developer/app-router/installation.mdx

```diff
@@ -15,15 +15,26 @@ The fastest way to get started with Makeswift on a new Next.js project is to fol
 [quickstart](/developer/quickstart) guide. If you have an existing Next.js application
 or want to set things up yourself, continue with the rest of this guide.
 
+<Note>
+  This installation guide is built for [Next.js
+  15](https://nextjs.org/blog/next-15) but also works with [Next.js
+  13.4](https://nextjs.org/blog/next-13-4) and later versions that use [App
```

Comment:
Maybe we want to also mention Next.js v14? As it is right now it feels like we're calling out a very old version. Maybe we can say "but also works with Next.js v13 and v14"?